prepare_intel.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+mkdir -p ./external/mkl/
+cp prepare_mkl.sh ./external/mkl/
+./external/mkl/prepare_mkl.sh 0
+cp -f tpp.h external/mkl/mklml_lnx_2017.0.0.20160801/include/mkl.h
+cp -f config.mk.tmp make/config.mk
```

Comment:
code these as conditions in makefile.
config.mk in root folder will override make/config.mk.
This approach won't work for existing isntallations